src/cms/views/users/region_user_view.py

```diff
@@ -92,46 +89,36 @@ def post(self, request, *args, **kwargs):
 
         # filter region users to make sure no users from other regions can be changed through this view
         user_instance = region.users.filter(id=kwargs.get("user_id")).first()
-        user_profile_instance = user_instance.profile if user_instance else None
 
         region_user_form = RegionUserForm(data=request.POST, instance=user_instance)
-        user_profile_form = RegionUserProfileForm(
-            data=request.POST, instance=user_profile_instance
-        )
-        user_profile_form.instance.region = region
 
-        if not region_user_form.is_valid() or not user_profile_form.is_valid():
+        if not region_user_form.is_valid():
             # Add error messages
             region_user_form.add_error_messages(request)
-            user_profile_form.add_error_messages(request)
-        elif not (
-            user_profile_form.cleaned_data.get("send_activation_link")
```

Comment:
True. By accident I removed some checks in the first place and then reintroduced them. It seems I missed this line. Fill fix it.

Reply:
But since the `RegionUserForm` inherits from `UserForm`, this will now be checked on calling `is_valid()`, so you can remove the explicit check in lines 97-106.